src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -205,110 +209,147 @@ export class SortFilterBar
   }
 
   handleResize(entry: ResizeObserverEntry): void {
-    if (entry.target === this.desktopSortSelector) {
-      this.desktopSelectorBarWidth = entry.contentRect.width;
+    if (entry.target === this.desktopSortContainer) {
+      this.desktopSortContainerWidth = entry.contentRect.width;
     } else if (entry.target === this.sortSelectorContainer) {
       this.selectorBarContainerWidth = entry.contentRect.width;
     }
   }
 
   private get sortDirectionSelectorTemplate() {
     return html`
-      <div id="sort-direction-selector">
-        <button
-          id="sort-ascending-btn"
-          class="sort-button ${this.sortDirection === 'asc' ? 'selected' : ''}"
-          ?disabled=${this.selectedSort === 'relevance'}
-          @click=${() => {
-            this.setSortDirections('asc');
-          }}
-        >
-          ${sortIcon}
-        </button>
-        <button
-          id="sort-descending-btn"
-          class="sort-button ${this.sortDirection === 'desc' ? 'selected' : ''}"
-          ?disabled=${this.selectedSort === 'relevance'}
-          @click=${() => {
-            this.setSortDirections('desc');
-          }}
-        >
-          ${sortIcon}
-        </button>
+      <button
+        class="sort-direction-selector"
+        ?disabled=${this.selectedSort === 'relevance'}
+        @click=${this.toggleSortDirection}
+      >
+        ${this.sortDirectionIcon}
+      </button>
```

Comment:
The sort direction selector is now a single button that displays the whole up/down arrow icon, rather than two separate triangle buttons.